js/src/utils.js

```diff
@@ -1,28 +1,8 @@
 
-var clone_deep = function(obj) {
-    return JSON.parse(JSON.stringify(obj))
-}
-
-/*
-    Object to avoid infinte loops when two objects van update eachother
-*/
-var Lock = function() {
```

Comment:
Also not sure about this one, maybe it was a leftover?

Reply:
yes, that's fine.